pegged/docs/wiki

```diff
@@ -1 +1 @@
-Subproject commit f111de4928c41cc22d2d093e3362f00bf7f80733
+Subproject commit 69188d442c1e7e4ad3976e5fc3988792723fc097
```

Comment:
First I changed it offline in a local checkout of the wiki source as a git subproject. Then I realised that the wiki isn't meant to be used like that.

I just edited the wiki online: https://github.com/PhilippeSigaud/Pegged/wiki/Extended-PEG-Syntax#case-insensitive-literals